tests/e2e/features/createSpace.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Create a space
+    As a super user
+    I want to create a space
+    So that I can add recipes and share my recipes
+
+
+    Background:
+        Given the user has signed up with the following details
+            | username         | superuser     |
+            | password         | super1234 |
+            | confirm_password | super1234 |
+        And the user has browsed to login page
+        And the user has logged in with username "superuser" and password "super1234"
+
+
+    Scenario: user creates a space
+        When the user creates a space "user's Space"
+        Then the user should redirect to search page
+        And the user should get success message "You have successfully created your own recipe space. Start by adding some recipes or invite other people to join you."
+        And user space "user's Space" should be visible on option menu
```

Comment:
IMO, these two steps should be enough to assert the search page. But I really don't know how the flow is.
```suggestion
        Then the user should get success message "You have successfully created your own recipe space. Start by adding some recipes or invite other people to join you."
        And user space "user's Space" should be visible on option menu
```